api/app/models/resource.rb

```diff
@@ -180,4 +190,12 @@ def split_iframe_dimensions
     iframe_dimensions.split("x")
   end
 
+  def variant_thumbnail_remote_url=(url_value)
+    self.variant_thumbnail = URI.parse(url_value)
+    @variant_thumbnail_remote_url = url_value
+
+  rescue OpenURI::HTTPError
```

Comment:
We need to do more than log this. We need to store a record of the failure to ensure that the fetcher not get caught in an infinite loop trying to refetch things that have failed more than one or two times.